src/Libraries/CoreNodes/String.cs

```diff
@@ -38,6 +38,24 @@ public static object ToNumber(string @string)
             throw new ArgumentException(Properties.Resources.StringToNumberInvalidNumberMessage, "str");
         }
 
+        /// <summary>
+        /// Get all of the number strings from the target string as a string
```

Comment:
By reading the code, you are actually returning the first number string, right? I guess I was referring to if it's worth to return a list of them if there are multiple numbers

Reply:
It returns all numbers, including if they have spaces or more letters inside the string, the numbers will be merged.

Reply:
![image](https://user-images.githubusercontent.com/31106432/144170042-aa6dd2ab-03dc-40a7-9d2f-875374d74599.png)

Reply:
Actually we still need the first or last zero in the string to do a lot of things, so I had to return the string result of the number